narwhal/node/src/main.rs

```diff
@@ -248,15 +248,6 @@ async fn run(
     worker_keypair: NetworkKeyPair,
     registry: Registry,
 ) -> Result<(), eyre::Report> {
-    // Only enabled if failpoints feature flag is set
-    let _failpoints_scenario: fail::FailScenario<'_>;
-    if fail::has_failpoints() {
-        warn!("Failpoints are enabled");
-        _failpoints_scenario = fail::FailScenario::setup();
```

Comment:
So the plan is to leave these in the code long term?

Reply:
Correct